### ISSUE TITLE
Atom HTML-Email Linter

### ISSUE THREAD

**Issue by coliff** [sindresorhus/module-requests#21]
Coding HTML emails can be a pain. Outlook uses Microsoft Word to render HTML emails, CSS support is very poor and there's lots of tips and tricks and things to avoid.

I found an online service: http://emaillint.com/ but a linter for code editors would be a brilliant thing.
I don't know how to go about creating a Linter for Atom, but if someone else created it I think I'd be able to open pull requests with additional rules to make it better.

**Comment by sindresorhus** [comment]
Accessing an Internet service for a linter would be too slow and not work offline. What needs to be done before an Atom plugin is a offline node module for linking HTML emails, that the Atom plugin can depend on.

**Comment by coliff** [comment]
Yes, I meant that it should work 100% offline and not rely on any Internet service.
Reference:
https://github.com/lukeschafer/EmailLint/tree/master/src

**Comment by Pentiado** [comment]
A long time ago after creating many emails for the client I needed to go through therapy which would bring the joy of coding back again and I build https://github.com/Pentiado/catch-me
![687474703a2f2f7333302e706f7374696d672e6f72672f35776c6574373867782f53637265656e5f53686f745f323031345f30355f31325f61745f32335f31355f30382e706e67](https://cloud.githubusercontent.com/assets/1911020/10776266/164b1f6a-7d10-11e5-83b1-13d8274046d7.png)
